pyproject.toml

```diff
@@ -85,6 +85,9 @@ sphinx = "*"
 pydata-sphinx-theme = "*"
 sphinx_design = "*"
 
+[tool.poetry.extras]
+dev = ["pytest", "pytest-xdist", "pytest-cases", "pytest-cov", "notebook", "jupyterlab", "pre-commit", "ruff", "snakeviz", "mypy", "memory-profiler", "sphinx", "pydata-sphinx-theme", "sphinx_design"]
+
```

Comment:
Changing this to use the "extras" feature, so can be installed via `pip install malariagen_data[dev]`.